pcsx2/gui/FrameForGS.cpp

```diff
@@ -623,6 +641,28 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 	out << std::fixed << std::setprecision(2) << fps;
 	OSDmonitor(Color_StrongGreen, "FPS:", out.str());
 
+	const u64 &smode2 = *(u64 *)PS2GS_BASE(GS_SMODE2);
+	int interlacing = (smode2 & 1) ? 1 : 0;      // INT: 0 = Progressive, 1 = Interlaced
+	int interlacing_mode = (smode2 & 2) ? 1 : 0; // FFMD: 0 = Field mode, 1 = Frame mode
```

Comment:
The ternary operator ``? 1 : 0`` is redundant here, coverity will complain about it.

Reply:
I only added it as a placeholder because VS was complaining about it(don't know why, building it now without ? 1 : 0 VS no longer mentions any issue :/). I pushed it because git had messed with my other PR and I wanted to know everything was fine with this one.